src/components/map/map.jsx

```diff
@@ -0,0 +1,17 @@
+import style from './map.module.scss';
```

Comment:
Please name all React-components with the big letter - in this case `Map` instead of `map`.
But make sure we have a Map class in JavaScript on global level, so it's better to named it somehow differently.